commercetools/resource_category_test.go

```diff
@@ -0,0 +1,165 @@
+package commercetools
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCategoryCreate_basic(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCategoryConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"commercetools_category.accessories", "name.en", "accessories",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_category.accessories", "description.en", "Standard description",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_category.accessories", "slug.en", "accessories",
+					),
+				),
```

Comment:
For completeness' sake could you expand the checks to include all settable fields?